Wspollokator/Views/Settings.swift

```diff
@@ -0,0 +1,203 @@
+//
+//  Settings.swift
+//  Wspollokator
+//
+//  Created by Piotr Czajkowski on 12/11/2021.
+//
+
+import SwiftUI
+
+struct Settings: View {
+    //@EnvironmentObject user
+    //Zmienne dotyczące customowych alertów
+    @State var isPasswordAlertPresented: Bool = false
+    @State var isDescriptionAlertPresented: Bool = false
+    //Zmienne do customowych alertów
+    @State var oldPassword: String = ""
+    @State var newPassword: String = ""
+    @State var description: String = UserProfile_Previews.users[0].description
+    //"Freezowanie" przycisków
+    @State private var isPhotoButtonClickable: Bool = true
+    @State private var isPasswordButtonClickable: Bool = true
+    @State private var isDescriptionButtonClickable: Bool = true
+    @State private var isLogOutButtonClickable: Bool = true
+    //Alerty
+    @State private var isPassPassedAlertPresented: Bool = false
+    @State private var isPassFailedAlertPresented: Bool = false
+    @State private var isLogOutAlertPresented: Bool = false
+    //Liczba nideudanych prób wpisania hasła
+    @State private var failedAttempts: Int = 0
+    //Dla failedAttempts = 5
+    @State private var isPassworButtonBlocked: Bool = false
+    //testing password, temporary
+    let password: String = "1234"
+    var user : User
+    var body: some View {
+        NavigationView {
+            ZStack {
+                List {
+                    HStack {
+                        Spacer()
+                        VStack {
+                            Avatar(image: user.avatarImage, size: 80)
+                            Button {
+                                if isPhotoButtonClickable{
+                                    //Image Picker prawdopodobnie
+                                    print("Zmienianie zdjęcia...")
+                                }
+                            } label: {
+                                Text("Zmień zdjęcie")
+                            }
+                        }
+                        Spacer()
+                    }
+                    Section {
+                        HStack {
+                            Text("Imię")
+                            Spacer(minLength: 50)
+                            Text(user.name)
+                        }
+                        HStack {
+                            Text("Nazwisko")
+                            Spacer(minLength: 50)
+                            Text(user.surname)
+                        }
+                        HStack {
+                            Text("Adres e-mail")
+                            Spacer(minLength: 50)
+                            Text("john.a....com ")
+                        }
```

Comment:
To powinny być pola, których wartości można modyfikować, jak np. w Ustawienia -> Ogólne -> To urządzenie... -> Nazwa (na iOS). Po kliknięciu w Nazwa przechodzi się do modyfikacji zawartości pola. Takie zachowanie można łatwo zrobić przy użyciu Form w SwiftUI.